### PR TITLE
Kan 6 harsha payment

### PR DESCRIPTION
**Pull request by Harsha1varthan**
#CheckOut page

AYS(1)

##Media Queries were Added

Usage:
`<BrowserRouter>
      <Routes>
        <Route path = "/checkout" element = {<CheckOut />}/>
      </Routes>
    </BrowserRouter>`

### CODE REVIEW

File: Frontend/src/App.css
Comment (aravind-digitech) [inline Frontend/src/App.css:1]: pls move all the changes to a css module in the component folder as in https://github.com/AYS-At-Your-Service/AYS/pull/13/files#diff-71ff0a73d0033410b9ff162a070eeb8a2f531418c2180812041b998676c3c4a4

File: Frontend/src/App.jsx
Comment (aravind-digitech) [inline Frontend/src/App.jsx:1]: pls revert changes in this file

File: Frontend/src/Payment/Checkout.jsx
Comment (aravind-digitech) [inline Frontend/src/Payment/Checkout.jsx:1]: pls import the css module and use it to apply styles like `inputStyles['search-box-container']` ref: https://github.com/AYS-At-Your-Service/AYS/pull/13/files#diff-4efee8ce3f0bbe6048ca338557b4d41369d3e6a059308a8907cbfecdde11e04f

File: Frontend/src/Payment/Checkout.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    <div className='CheckOut_Header'>
+          <h1>Checkout</h1>
+    </div>
+      <div className='Box-Wrapper'>
+        <div className = "Inside-Box">
+          <div style={{display: "flex", justifyContent: "space-between"}}>
+            <div style={{display: "flex"}}>
+                <img src = {Image} className="Image"></img>
+                <div>
+                  <h4>Send Booking Details</h4>
```
Comment (aravind-digitech) [inline Frontend/src/Payment/Checkout.jsx]: Use constants by importing a constants file like `{consts.STATICPLACEHOLDER}` ref: https://github.com/AYS-At-Your-Service/AYS/pull/13/files#diff-4efee8ce3f0bbe6048ca338557b4d41369d3e6a059308a8907cbfecdde11e04f

File: Frontend/src/Payment/Checkout.jsx
Comment (aravind-digitech) [inline Frontend/src/Payment/Checkout.jsx:1]: Components should be under Frontend/src/components

**Comment by aravind-digitech** [comment]
I guess latest changes are missing in this branch, pls create a new branch from Development and add your changes there

**Comment by Harsha1varthan** [comment]
Okay bro

**Review by aravind-digitech — CHANGES_REQUESTED** [review]

**Comment by aravind-digitech** [comment]
Re raised new PR hence closing this one

**Pull request closed without merging**